### ISSUE TITLE
m_rawinput 1 and VGUI menu 

### ISSUE THREAD

**Issue by Drone101** [ValveSoftware/halflife#815]
I love the command but in CS 1.6 setting "m_rawinput 1" means the game retains your mouse's position between VGUI menus and from one round to the next when buying.

This is a massive change to the behaviour of the mouse in VGUI menus.

With "m_rawinput 0" the mouse centres to the middle of your crosshair after you complete a purchase.
With "m_rawinput 1" it remembers it's position from one purchase to the next.

The consequence of which is that you can no longer use pure muscle memory to make each individual purchase.

As a long standing cs player I am used to buying items without concious thought. With the mouse position no longer resetting each round purchase takes much longer as you have to consciously reposition the mouse. When you have only 3 seconds to call a strat and buy up in a league match this is incredibly frustrating.

Suggested Solution 1: Remove the way the new "m_rawinput 1" remembers your mouse position each round or when using the mouse in other menus in-game
Suggested Solution 2: Create new variable "m_mousefocusreset 1" to keep the buy menus working as we have trained ourselves for.

Related to but separate from:  <a href=https://github.com/ValveSoftware/halflife/issues/150> [HL1/Mods] Mouse Focus gets lost when using VGUI menus or click scoreboard. </a>
 (Exact opposite problem!)

TL:DR CS 1.6 VGUI buy menus relied on mouse resetting to let you buy from memory with a flick of the mouse, the same flick each round would get you the same gun. With "m_rawinput 1" as the mouse focus (position) is retained you now have to actively look where your mouse is before and during buying - wasting time at a very critical moment in a round. Thanks for your help, really appreciate the love you guys are giving to the goldsource engine!
